### ISSUE TITLE
Test this project with Symfony 4.4 and 5.0

### ISSUE THREAD

**Issue by javiereguiluz** [symfony/demo#987]
**Description of the idea**

Symfony 4.4 and 5.0 will be released at the end of November 2019. A new major version is always very important in Symfony timeline, so let's test all features in those versions too.

**The work to do**

The `.travis.yml` file at the root of the project must be updated to also test the project with 4.4 and 5.0. These new test jobs must also be added to the list of "things allowed to fail" ... because we don't care if they fail today ... we only want to closely check those errors to see if Symfony breaks things.

Thanks!

**Comment by mazarini** [comment]
Hi,

I made a try but it fail :
```
$ composer $ACTION
Loading composer repositories with package information
Updating dependencies (including require-dev)
PHP Fatal error:  Allowed memory size of 1610612736 bytes exhausted (tried to allocate 4096 bytes) in phar:///home/travis/.phpenv/versions/7.3.2/bin/composer/src/Composer/DependencyResolver/RuleSetGenerator.php on line 129
Fatal error: Allowed memory size of 1610612736 bytes exhausted (tried to allocate 4096 bytes) in phar:///home/travis/.phpenv/versions/7.3.2/bin/composer/src/Composer/DependencyResolver/RuleSetGenerator.php on line 129
The command "composer $ACTION" failed and exited with 255 during .
```
$ACTION=update
The same fail with 4.0 and 5.0 (both minimum-stability=dev)

**Comment by javiereguiluz** [comment]
Fixed in #1007.
